plone/app/z3cform/converters.py

```diff
@@ -341,9 +348,9 @@ def toWidgetValue(self, value):
         return [IUUID(value)]
 
 
-@adapter(ICollection, IRelatedItemsWidget)
```

Comment:
same as above, shouldn't we additionally keep the `IRelatedItemsWidget` adapted converter for BBB, for those using the related items widget instead the new content browser?